docs/conf.py

```diff
@@ -106,22 +106,21 @@
 html_theme_options = {
     "github_url": APP_DATA["homepage"],
     "use_edit_page_button": True,
-    "navbar_start": ["launch.html"],
-    "navbar_center": ["navbar-logo.html", "navbar-nav.html"],
+    "navbar_start": ["navbar-logo", "version-switcher"],
     "icon_links": [
         {
             "name": "PyPI",
             "url": "https://pypi.org/project/jupyterlite",
             "icon": "fa-solid fa-box",
         },
-        {
-            "name": "GitHub",
-            "url": APP_DATA["repository"]["url"],
-            "icon": "fa-solid fa-github-square",
-        },
     ],
     "pygment_light_style": "github-light",
     "pygment_dark_style": "github-dark",
+    "logo": {
+        "alt_text": "JupyterLite",
+        "image_light": "_static/wordmark.svg",
+        "image_dark": "_static/wordmark-dark.svg",
```

Comment:
Not sure why this is not picked up when using the dark theme:

![image](https://github.com/jupyterlite/jupyterlite/assets/591645/aae3a02c-d4c8-4cae-a82f-89de6de52770)

Reply:
Will open an issue as a follow-up.

Reply:
Opened https://github.com/jupyterlite/jupyterlite/issues/1166